demo/chatbot_multimodal/run.py

```diff
@@ -5,22 +5,21 @@ def add_text(history, text):
     history = history + [(text, text + "?")]
     return history
 
-def add_image(history, image):
-    history = history + [(f"![](/file={quote(image.name)})", "Cool pic!")]
+def add_file(history, file):
+    history = history + [(file.name, None)]
     return history
 
 with gr.Blocks() as demo:
-    chatbot = gr.Chatbot(elem_id="chatbot").style(height=500)
+    chatbot = gr.Chatbot(elem_id="chatbot").style(height=750)
     
     with gr.Row():
         with gr.Column(scale=0.85):
             txt = gr.Textbox(show_label=False, placeholder="Enter text and press enter, or upload an image").style(container=False)
         with gr.Column(scale=0.15, min_width=0):
-            btn = gr.UploadButton("🖼️", file_types=["image"])
-            
+            btn = gr.UploadButton("🖼️", file_types=["image", "video", "audio"])
```

Comment:
```suggestion
            btn = gr.UploadButton("📁", file_types=["image", "video", "audio"])
```
since we now accept multiple files!